sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Listeners/ServiceBusListener.cs

```diff
@@ -18,6 +19,8 @@
 using Microsoft.Azure.WebJobs.Host.Listeners;
 using Microsoft.Azure.WebJobs.Host.Scale;
 using Microsoft.Extensions.Logging;
+using Microsoft.Azure.Amqp.Framing;
+using Newtonsoft.Json.Schema;
```

Comment:
<img src="https://media3.giphy.com/media/26hkhKd2Cp5WMWU1O/giphy.gif"/>

Reply:
<img src="https://media4.giphy.com/media/xTiTnqUxyWbsAXq7Ju/giphy.gif"/>

Reply:
> Ah oh no that was not on purpose

Hah yeah I figured it was a remnant from some Schema Registry investigations.